weather_forecast/weather_forecast.py

```diff
@@ -0,0 +1,50 @@
+import datetime
+import requests
+import pyttsx3
+import datetime
+import pyaudio
+import speech_recognition as sr
+
```

Comment:
Remove imported libraries `pyttxs3` 2nd `datetime` `pyaudio` and `speech_recognition`